.github/workflows/e2etests.yaml

```diff
@@ -0,0 +1,127 @@
+      provider: Local 
+      sm_version: ${{ needs.SetPatternfileForServicemeshInstall.outputs.sm_version }}
+      adapter_version:  ${{ needs.SetPatternfileForServicemeshInstall.outputs.adapter_version }}
+      output_filename: data.json
+    secrets:
+      token: ${{ secrets.PROVIDER_TOKEN }}  
+
+  UpdateDocsForServicemeshInstall:
+    needs: TestAdapterForServicemeshInstall
+    if:  "always() && github.event_name != 'pull_request' "
+    runs-on: ubuntu-latest
+    steps:
+      - name: Skip if needed
+        run: |
+            echo "this is it ${{ needs.SkipIfNeeded.outputs.skipdocupdate }} "
+            if [ "${{github.event_name }}" == "push" ];then
+              echo "version=edge" >> $GITHUB_ENV
+            fi
+            if [ "${{github.event_name }}" == "release" ];then
+              echo "version=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
+            fi
+      - uses: actions/checkout@master
+        with:
+          repository: meshery/meshery 
+          token: ${{ secrets.GH_ACCESS_TOKEN }}   
+      - name: DownloadJSON
+        uses: actions/download-artifact@v2
+        with:
+          name: data.json
+      - name: Add commit SHORT_SHA
+        run: echo "SHORT_SHA=`echo ${GITHUB_SHA} | cut -c1-8`" >> $GITHUB_ENV
+      - name: echo results
+        run: |
+          pretty=$(jq . data.json)
+          rm data.json
+          mkdir -p ./docs/_compatibility/meshery-consul
+          cd ./docs/_compatibility/meshery-consul
+          touch data.json
+          echo $pretty > data.json
+          cat data.json
+          touch test.md
+          filename=(`date +%m-%d-%Y-%H-%M-%m_${SHORT_SHA}`)
+          echo $filename
+          echo "---
+          timestamp: $(jq '.metadata.runon| strptime("%a %b %e %X %Z %Y") | strftime("%Y-%m-%d %X %Z %a")' data.json)
+          meshery-component: meshery-consul
+          meshery-component-version: $version
+          meshery-server-version: $(jq '.metadata."meshery-server-version"' data.json)
+          k8s-distro: minikube
+          k8s-version: $(jq '.metadata.k8s_version' data.json)
+          service-mesh: Consul
+          service-mesh-version: $(jq '.metadata.service_mesh_version' data.json)
+          tests:
+            consul-connect-injector-webhook-deployment: $(jq '.pods_status."consul-connect-injector-webhook-deployment"' data.json)
+            consul-controller: $(jq '.pods_status."consul-controller"' data.json)
+            consul-server:  $(jq '.pods_status."consul-server"' data.json)
+            consul-webhook-cert-manager:  $(jq '.pods_status."consul-webhook-cert-manager"' data.json)
+            consul:  $(jq '.pods_status."consul"' data.json)
+            prometheus-server:  $(jq '.pods_status."prometheus-server"' data.json)
+          overall-status: $(jq '."overall-status"' data.json)
+          ---" >  test.md
+          mv test.md $filename.md
+          rm data.json
+      - name: Commit
+        uses: stefanzweifel/git-auto-commit-action@v4
```

Comment:
It is this plugin necessary to use ? Maybe simple git command with those parameters would be enough?

Reply:
Yes, It saves us the burden of writing the script ourselves. @AugustasV